addon/components/rdfa-date-plugin/card.hbs

```diff
@@ -38,6 +38,7 @@
             />
           {{/if}}
         </AuFormRow>
+        <AuButton @icon="info-circle" {{on 'click' this.toggleHelpModal}}></AuButton>
```

Comment:
I really don't like the look of this button
I'd suggest using the "link-secondary" skin and putting it next to the custom date control

Reply:
An au-badge (https://appuniversum.github.io/ember-appuniversum/?path=/story/components-notifications-aubadge--component) is maybe also another possibility.